src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocmethod/InputJavadocMethod3.java

```diff
@@ -6,7 +6,7 @@
 
 package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocmethod;
 
-// ok
+
```

Comment:
<-- unrelated to this line -->
You seem to be facing the same trouble again as mentioned at https://github.com/checkstyle/checkstyle/pull/14460#pullrequestreview-1874295173